api/db/seeds/data/team-certification/cases/simple-pro-certification.js

```diff
@@ -165,6 +165,11 @@ export class ProSeed {
       normalizeStringFnc: normalize,
     });
 
+    await sessionManagementUseCases.authorizeCertificationCandidateToStart({
```

Comment:
Ca a pas l'air comme cela mais ce point la montre un defaut de notre code PRO que les seeds SCO m'ont permis de voir (car ils utilisent pas le meme usecase d'enrolment) :

**le usecase addCandidateToSession met a jour en BDD des attributs dont il ne devrait pas se soucier** en l'occurence le authorized-to-start, car il fait confiance au model qu'il recoit par le serializer.

Par contre j'ai verifie un truc important : le joi nous protege, dont on peut pas utiliser une requete modifiee sur un front pour modifier via la console le authorize-to-start sauvagement. Mais on a quand meme un petit soucis de cloisonnement sur les responsabilites du usecase je trouve 😅